packages/alfa-style/src/element/helpers/get-bounding-box.ts

```diff
@@ -0,0 +1,18 @@
+import { Device } from "@siteimprove/alfa-device";
+import { Element } from "@siteimprove/alfa-dom";
+import { Option } from "@siteimprove/alfa-option";
+import { Rectangle } from "@siteimprove/alfa-rectangle";
+
+/**
+ * @public
+ * Gets the bounding box, corresponding to a specific device, of an element
+ *
+ * @privateRemarks
+ * We don't use the passed in device yet, but later we should use it to ensure the device used to collect the bounding box corresponds to the current device
+ */
+export function getBoundingBox(
+  element: Element,
+  device: Device
```

Comment:
**Suggestion:** This should also accept a `Context` parameter (defaulting to `Context.empty()`), and return `None` if it is not `Context.empty` (since we assume layout is grabbed on empty contexts).